iOS/ViewControllers/Courses/PeerAssessmentViewController.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Common
+import UIKit
+
+class PeerAssessmentViewController: UIViewController {
```

Comment:
Similar to the `LTIHintViewController`, this view controller should be named `PeerAssessmentHintViewController` as we do not cover a peer assessment with this view but only a page before the peer assessment.